canonical-web-and-design/vanilla-framework
# Add component menu

## Done

Add `component_tabs.yaml` file that stores the /docs tab menu. It will be the source of truth for when the menu shows up on each docs page and which tab items get displayed (design/accessibility).

Structure of the `component_tabs.yaml`:
```yaml
patterns/accordion:
  accessibility: True
  design_guidelines: False 
```

This PR add the menu only for the accordion docs page, future pages will be added in a separate PR.

## QA

- Open https://vanilla-framework-4414.demos.haus/docs 
- Check the side menu is highlighting the right link
- Go to https://vanilla-framework-4414.demos.haus/docs/accordion
- Check the side menu is highlighting the right link
- Is the tab menu there?
- Go to https://vanilla-framework-4414.demos.haus/docs/patterns/accordion/accessibility
- Check the side menu is highlighting the right link
- Check the tab menu is highlighting the right link

## Screenshots

![image](https://user-images.githubusercontent.com/6387619/163981817-380741b2-b351-4ce2-89b0-6f56b4b094eb.png)
